atlas_metadata_validator/atlas_checker.py

```diff
@@ -177,16 +182,19 @@ def run_singlecell_checks(self, logger):
                 logger.error("Required SDRF field \"{}\" not found.".format(field))
                 self.errors.add("SC-E05")
 
-        # Require at least one of those fields to be present
-        required_download_fields = get_controlled_vocabulary("raw_data_download_sdrf_fields", "atlas", logger)
-        found_download_field = False
-        for field in required_download_fields:
-            if field.lower() in self.sdrf_values or get_name(field) in self.header_dict:
-                found_download_field = True
-                break
-        if not found_download_field:
-            logger.error("Required data download SDRF field \"{}\" not found.".format(" or ".join(required_download_fields)))
-            self.errors.add("SC-E12")
+        # Require at least one of those raw data fields to be present except for AnnData experiments
+        if self.is_anndata:
+            logger.debug("Found AnnData ingested experiment, no raw data SDRF fields required")
```

Comment:
I would change "logger.debug" for "logger.info"

Reply:
done!